services/cloudapi/cloudapiserver/server.go

```diff
@@ -0,0 +1,211 @@
+package cloudapiserver
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	ionoscloud "github.com/ionos-cloud/sdk-go/v6"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services/cloudapi"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils"
+)
+
+type Service struct {
+	Client *ionoscloud.APIClient
+	Meta   interface{}
+	D      *schema.ResourceData
+}
+
+var SuspendCubeLast error
+
+const (
```

Comment:
use camelcase please

Reply:
this error is exported to signal to the caller that we will suspend the cube server after all other updates have been applied,
it is mostly for making the intent more explicit, it could be replaced with simply returning nil from the `UpdateVmState` function

Reply:
Sorry, I meant below, for the all the constants. CubeServerType

Reply:
renamed to ErrSuspendCubeLast